sport/app/football/views/fragments/matchSummary.scala.html

```diff
@@ -16,15 +16,15 @@
         "match-summary--responsive" -> responsive
     ))" data-match-id="@theMatch.id">
     @if(theMatch.isLive || theMatch.isResult){
-        <h1 class="u-h">@homeTeam.name @homeTeam.score - @awayTeam.score @awayTeam.name</h1>
+        <h1 class="u-h">@model.CompetitionDisplayHelpers.correctTeamName(homeTeam.name) @homeTeam.score - @awayTeam.score @model.CompetitionDisplayHelpers.correctTeamName(awayTeam.name)</h1>
```

Comment:
you could @import model.CompetitionDisplayHelpers.correctTeamName

Reply:
Done :)